lib/commands/command.bash

```diff
@@ -9,7 +9,14 @@ fi
 
 # Load direnv stdlib if not already loaded
 if [ -z "$(declare -f -F watch_file)" ]; then
-  eval "$(asdf exec direnv stdlib)"
+  # try using direnv from PATH if already available, since
+  # this is much faster than going thru asdf.
+  # Otherwise for non-system direnv we take the slow path.
+  if command -v direnv >&2>/dev/null; then
```

Comment:
Shall we always try using system direnv ? or only if specified via an environment variable `ASDF_DIRENV_BIN` as suggested in https://github.com/asdf-community/asdf-direnv/issues/80#issuecomment-1079478125 


Having a direnv executable reference could be a noticeable performance benefit, either a system or the resolved path to an asdf-installed one. Perhaps we could have something like this on README ? Do you feel like having yet another configuration variable makes things more complicated?

```
export ASDF_DIRENV_BIN="$(command -v direnv)" # or $(asdf which direnv) 
```

Reply:
Improved how we determine which `direnv` exectuable to use:

https://github.com/asdf-community/asdf-direnv/pull/120/files#diff-2e890d79aeb124a20bc085e927f3c5a70e07319924cdf1870e6bac5c26347efdR36

Still have to improve documentation about these `ASDF_DIRENV_BIN` variable, since our README.md assumed most people wanted to use an asdf-installed direnv. However many people actually have a system-managed version.

Reply:
Since this is a backward compatible change, perhaps we could merge this if code looks good to reviewers, and improve the README intructions and usage ergonomics on other branch.